src/routes/messages.js

```diff
@@ -16,10 +16,11 @@ module.exports = (() => {
     *
 	* @apiHeader {String} token Authorization Bearer Token.
     * @apiParam {String} mid Message's id.
+	* @apiParam (Request body) {String} emoji Emoji reaction.
     *
-    * @apiSuccess (Created 201) {Number} id Message ID.
-    * @apiSuccess (Created 201) {Number} id User ID.
-    * @apiSuccess (Created 201) {String} emoji Emoji.
+    * @apiSuccess (Created 201) {Number} messages_id Message ID.
+    * @apiSuccess (Created 201) {Number} users_id User ID.
+    * @apiSuccess (Created 201) {String} emoji Emoji reaction.
     *
```

Comment:
I think these are the names you used according to the example response so I just updated 'em here.